docs/usage/plugin.md

```diff
@@ -9,7 +9,7 @@ description: >
 The following plugins are used within the example
 
 * [Kaniko](/docs/usage/plugins/registry/Kaniko.md)
-* [Vault](/docs/usage/tour/vault secrets.md)
```

Comment:
this plugin is currently commented out (ie. no doc page is being generated for it)